src/OSPSuite.UI/Views/Importer/ColumnMappingControl.cs

```diff
@@ -104,7 +109,7 @@ public override void InitializeBinding()
          base.InitializeBinding();
          _gridViewBinder.AddUnboundColumn()
             .WithCaption(UIConstants.EMPTY_COLUMN)
-            .WithShowButton(ShowButtonModeEnum.ShowOnlyInEditor)
+            .WithShowButton(ShowButtonModeEnum.ShowAlways)
```

Comment:
Button should always be visible

Reply:
in fact this is only required for combo box. Otherwise buttons are visible by default